src/synopsys-action/tools-parameter.ts

```diff
@@ -315,6 +298,8 @@ export class SynopsysToolsParameter {
       covData.data.network = {airGap: parseToBoolean(inputs.ENABLE_NETWORK_AIR_GAP)}
     }
 
+    covData.data.coverity = Object.assign({}, this.setCoverityArbitraryArgs(), covData.data.coverity)
```

Comment:
This will replace the child object with Parent Object  (Arbitrary) and we will lose data prepared to covData.data.coverity.